packages/tessellate-request/lib/configuration/oauth-configuration-file-loader.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import nconf from 'nconf'
+import fs from 'fs'
+import type {OAuthConfig} from './oauth-configuration'
```

Comment:
Please use spaces here and in the other import statements, i.e. `{ OAuthConfig }`